auto_instrument_handler/index.js

```diff
@@ -0,0 +1,26 @@
+const lumigo = require('@lumigo/tracer')({});
+
+const ORIGINAL_HANDLER_KEY = 'LUMIGO_ORIGINAL_HANDLER';
+
+const handler = (event, context, callback) => {
+  const originalHandler = process.env[ORIGINAL_HANDLER_KEY];
+  if (!originalHandler) {
+    throw Error(
+      'Could not load the original handler. Are you sure that the handler is correct?'
+    );
+  }
+  const handlerParts = originalHandler.split('.');
+  const moduleName = `/var/task/${handlerParts.slice(0, -1).join('.')}`;
+  const functionName = handlerParts.slice(-1);
+  let module;
+  try {
+    module = require(moduleName);
+  } catch (e) {
+    throw Error(
```

Comment:
log in debug the moduleName you tried to find (i know you don't have the logger here) maybe just check the env var and print, your call

Reply:
I'll put it inside the error message 👍